_posts/2019-06-05-hibernate-search-extension.adoc

```diff
@@ -0,0 +1,124 @@
+---
+layout: post
+title: 'New extension: Hibernate Search + Elasticsearch'
+date: 2019-06-05
+tags: quarkus hibernate elasticsearch
+author: gsmet
+---
+
+In the 0.15.0 version of Quarkus, we introduced the Hibernate Search + Elasticsearch extension and, just a few days ago, we added link:/guides/hibernate-search-guide[a guide] and https://github.com/quarkusio/quarkus-quickstarts/tree/master/hibernate-search-elasticsearch[a quickstart] explaining how to use it.
+
+Note that this extension is still experimental as it is based on an Alpha version of Hibernate Search 6.
+While the code is stable and thoroughly tested, some features are still missing and we might change some APIs along the way to the final version - even if the Hibernate team has made a great effort to stabilize them before the inclusion in Quarkus.
```

Comment:
I'd put the whole experimental aspect down toe blog.
Or consider that Quarkus is a 0.x and everything is experimental to some extends. 
Come to think of it, I'd make sure people want to try in the blog and have the guide be the cautious provider.